linode/manager
# fix: [M3-5970] - Fix alignment issues with Node Pool table and buttons

## Description 📝

Fixes alignment and spacing issues with Node Pools table and action buttons in table header component. The initial ask for this ticket was to fix the alignment for the Delete Cluster button. However, after investigation it was deemed an implementation flaw since the button is styled to look like a link, giving it a white background and causing the text to look misaligned. 

## Preview 📷

**Before:**
![a](https://user-images.githubusercontent.com/119514965/230514303-441fe6ce-2d10-405a-91e3-7633e083d066.jpg)
![b](https://user-images.githubusercontent.com/119514965/230514306-6ebea5cb-8bc9-440f-8c61-0926d8741bbd.jpg)


**After:**
![c](https://user-images.githubusercontent.com/119514965/230514317-f9160c24-5777-4b00-b90b-a906bee31a00.jpg)
![d](https://user-images.githubusercontent.com/119514965/230514324-c2c8d1c9-aa7d-4cf1-98b3-bb7e0493bb90.jpg)

## How to test 🧪
Navigate to the Kubernetes Summary view page and inspect the Node Pools table component. Verify that the Add A Node Pool button does not have any extra margin on its right hand side.

Next, reduce the viewport width and inspect that the row elements wrap around correctly and align with each other. Also, notice the removal of the extra white space around the table component.
